docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java

```diff
@@ -130,7 +130,8 @@ private static CubeContainer createVncContainer(final Path dockerVolume, String
         // Using sleeping strategy since VNC client is a CLI without exposing a port
         Await await = new Await();
         await.setStrategy("sleeping");
-        await.setSleepTime("100 ms");
+        await.setSleepTime("1 s");
```

Comment:
I've never seen the browser container start in under a second, so think this is a little more forgiving.